src/bin/zmcontrol

```diff
@@ -559,12 +560,49 @@ sub getHostName {
 }
 
 sub displayVersion {
+    my $zal_path =     "/opt/zextras/lib/ext/carbonio/zal.jar";
+    my $zextras_path = "/opt/zextras/lib/ext/carbonio/carbonio.jar";
+
+    my $additional_zal_version = "";
+    if (-e $zal_path and -e $zextras_path) {
+         $additional_zal_version = `carbonio core getVersion`;
+        chomp $additional_zal_version;
+    }
+
+    my $release = qx(cat /opt/zextras/.version);
+    chomp $release;
+
+    my $output = "Carbonio Release $release";
+    if (! ($additional_zal_version eq "")) {
+        $output .= "\nAdvance module version:\n" . $additional_zal_version
+    }
+    print "$output.\n";
+}
+
+sub displayPackagesVersion {
+    my $zal_path =     "/opt/zextras/lib/ext/carbonio/zal.jar";
+    my $zextras_path = "/opt/zextras/lib/ext/carbonio/carbonio.jar";
     my $release = qx(cat /opt/zextras/.version);
     chomp $release;
-    my $string = "Release $release";
-    $string .=
-      ( ( -f "/opt/zextras/bin/zmbackupquery" ) ? "" : " Community Edition" );
-    print "$string.\n";
+    my $distro = `cat /etc/os-release | grep ID= | head -n1 | cut -f2 -d'='`;
+
+    my $packages = "";
+    if ($distro eq "ubuntu") {
+        $packages = `dpkg-query -W -f'${Package} ${Version}\n' carbonio*`;
```

Comment:
This line has the following compilation errors:
```
Global symbol "$Package" requires explicit package name (did you forget to declare "my $Package"?) at /opt/zextras/bin/zmcontrol line 591, <DATA> line 960.
Global symbol "$Version" requires explicit package name (did you forget to declare "my $Version"?) at /opt/zextras/bin/zmcontrol line 591, <DATA> line 960.
Execution of /opt/zextras/bin/zmcontrol aborted due to compilation errors.

```

Reply:
Done my bad